package.json

```diff
@@ -113,6 +114,7 @@
     "vue-server-renderer": "~2.5.16",
     "vue-styleguidist": "~1.7.7",
     "vue-template-compiler": "~2.5.16",
+    "vue-test-utils": "^1.0.0-beta.11",
```

Comment:
Test Utilities and babel-plugin-transform-object-rest-spread should go into the dev section of dependencies

Reply:
Cool, thanks... forgot to move those... Should have added them with `yarn add --dev`

Reply:
Did you update the yarn lock by running yarn install again?

Reply:
Yes, although I didn't notice a difference...